### ISSUE TITLE
"Invalid byte sequence in conversion input" while using an emoji in a command

### ISSUE THREAD

**Issue by gagangulyani** [microsoft/vscode#121009]
- VS Code Version: 1.54.3
- OS Version: 5.9.16-1-MANJARO

Steps to Reproduce:

1. Go to VSCode terminal and execute the following command `notify-send -a "Reminder 🐍" -t 5000 -u "normal" "Blink"`
2. It will do the same while executing a Python script from VS Code which executes the same command.
3. While moving the cursor from left to right or vice-versa after navigating the previously executed command (via up-arrow key), characters get replaced with adjacent characters (weird issue) and the most right one remains there and isn't navigable.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by meganrogge** [comment]
Please provide a screenshot or [a gif](https://gifcap.dev/) illustrating the problem (attaching images won't work via email, you need to go to github.com). See [this wiki page](https://github.com/microsoft/vscode/wiki/Terminal-Issues) for more general info on how to report actionable terminal issues.
/label "needs more info" integrated-terminal

**Comment by gagangulyani** [comment]
Commands I executed in Terminal:

+ `echo "Hello 🐍"`
+ `notify-send -a "Reminder 🐍" -t 5000 -u "normal" "Blink"`

Also,
I tried to navigate to left and right using arrow keys on my keyboard, but it there's a weird glitch which can be seen in the gif attached below.

![recording](https://user-images.githubusercontent.com/34723580/114755916-ca8f7380-9d77-11eb-9d59-20fd3d50f66a.gif)

**Comment by meganrogge** [comment]
Does turning off the setting `terminal.integrated.gpuAcceleration `help?

**Comment by gagangulyani** [comment]
Nope, I tried that by adding `"disable-hardware-acceleration": true` in *argv.json* following [this link]("disable-hardware-acceleration": true).

Also, I tried the `--disable-gpu` command line flag as well.

Same result.

**Comment by gagangulyani** [comment]
Whenever I try to commit from gitlens, I'm facing another issue it redirects to my terminal for GPG key but my own terminal starts glitching as well.
